torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -129,30 +130,90 @@ class TensorExprFuser {
   TensorExprFuser(std::shared_ptr<Graph> graph, size_t min_group_size)
       : graph_(std::move(graph)), min_group_size_(min_group_size) {}
 
+  // TODO: handle better
```

Comment:
Nit: I'm not sure such TODO is helpful - could we either add more details or remove it altogether?